Modified for AZL to take into account other patches and config settings.
Specifically, removed the second '#' from the blank comment on line 15 of this file.


diff -up openssl-3.0.0/apps/openssl.cnf.xxx openssl-3.0.0/apps/openssl.cnf
--- openssl-3.0.0/apps/openssl.cnf.xxx	2021-11-23 16:29:50.618691603 +0100
+++ openssl-3.0.0/apps/openssl.cnf	2021-11-23 16:28:16.872882099 +0100
@@ -55,11 +55,11 @@ providers = provider_sect
 # to side-channel attacks and as such have been deprecated.
 
 [provider_sect]
-default = default_sect
+##default = default_sect
 ##legacy = legacy_sect
 #
-[default_sect]
-activate = 1
+##[default_sect]
+##activate = 1
 
 ##[legacy_sect]
 ##activate = 1
